clients/locationClient/locationClient.go

```diff
@@ -55,3 +55,29 @@ func GetLocationList() (LocationList, error) {
 
 	return locationList, nil
 }
+
+func GetLocation(location string) (*Location, error) {
+	if len(location) == 0 {
+		return nil, fmt.Errorf(azure.ParamNotSpecifiedError, "location")
```

Comment:
We have a few options on this - it's used throughout the codebase hence the original decision (I assume) to export it. We could either duplicate wherever necessary and make it internal or leave it as is. Any preference?